exchanges/huobi/huobi_wrapper.go

```diff
@@ -186,6 +186,17 @@ func (h *HUOBI) SetDefaults() {
 	h.WebsocketOrderbookBufferLimit = exchange.DefaultWebsocketOrderbookBufferLimit
 }
 
+// Bootstrap ensures that future contract expiry codes are loaded if AutoPairUpdates is not enabled
+func (h *HUOBI) Bootstrap(_ context.Context) (continueBootstrap bool, err error) {
```

Comment:
A much cleaner solution, thank you!
Would you consider adding this?
```go
func TestBootstrap(t *testing.T) {
	t.Parallel()
	h := new(HUOBI) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
	require.NoError(t, testexch.Setup(h), "Test Instance Setup must not fail")

	_, err := h.Bootstrap(context.Background())
	require.NoError(t, err)

	h.futureContractCodes = nil
	h.Features.Enabled.AutoPairUpdates = false
	_, err = h.Bootstrap(context.Background())
	require.NoError(t, err)
	require.NotNil(t, h.futureContractCodes)
}

```

Reply:
Fixed [f003ce6](https://github.com/gbjk/gocryptotrader/commit/f003ce690d0da9f05ae1f6ea591bfcafd199a878)

Immediately rebased down.

Reply:
Two lint niggles fixed immediately after this. Still no idea when and when govet doesn't like these shadows. It seems completely arbitrary.